src/main/java/redis/clients/jedis/args/LazyRaw.java

```diff
@@ -0,0 +1,18 @@
+package redis.clients.jedis.args;
+
+public class LazyRaw implements Rawable {
```

Comment:
Can you explain the name of the class I see the "raw" part as having a `byte[]` payload but I don't get the "lazy" part?

Reply:
@bsbodden All our existing implementation of `Rawable` are such that `getRaw()` can be called immediately to get the proper value. But here I needed to create an object but I would know the actual value later on. And, among all the names (something ending in Raw) that I could think of LazyRaw seemed least bad. Do you have any suggestion?